build/resources/main/application.properties

```diff
@@ -0,0 +1,8 @@
+server.port = 8081
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/wolox
+spring.datasource.username= postgres
```

Comment:
these sensible information should not be pushed in a repository, for these we can use envioment variables